pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1901,4 +1901,13 @@ self: super: {
 
   phonetic-languages-phonetics-basics = appendPatch super.phonetic-languages-phonetics-basics ./patches/phonetic-languages-phonetics-basics-haddock.patch;
 
+  Frames-streamly = overrideCabal (super.Frames-streamly.override { relude = super.relude_1_0_0_1; }) (drv: {
+    # https://github.com/adamConnerSax/Frames-streamly/issues/1
+    patchPhase = ''
+cat > example_data/acs100k.csv <<EOT
```

Comment:
I'd prefer to use `overrideSrc` and just fetch the source from github instead of using the sdist from hackage.

Reply:
could be an option, but there are no tags/releases on github and full file is several MB so thought a 2-line version would do